src/test/integration/IntegrationDeployer.t.sol

```diff
@@ -378,14 +385,20 @@ abstract contract IntegrationDeployer is Test, IUserDeployer {
                 tokenBalances[i] = balance;
                 strategies[i] = strat;
             }
-
-            return (strategies, tokenBalances);
         } else if (assetType == HOLDS_ETH) {
-            revert("_getRandAssets: HOLDS_ETH unimplemented");
+            strategies = new IStrategy[](1);
+            tokenBalances = new uint[](1);
+
+            // Award the user 32 ETH
+            uint amount = 32 ether;
```

Comment:
Should we award the user some value of ETH in the set {<32 ETH, 32 ETH, > 32ETH} ?

Reply:
Great question, I was wondering too. 

There is a slight hitch with just using random ETH values: `EigenPod.stake` requires that `msg.value == 32 ether`. Also, the deposit contract itself has a few requirements that I'd want the mock beacon chain to enforce:
* `depositValue >= 1 ether`
* `depositValue % 1 gwei == 0`
* `depositValue / 1 gwei <= type(u64).max`

The easy, immediate changes we could make:
* Have `BeaconChainMock.newValidator` enforce the deposit contract conditions I listed above
* Create users that are awarded some multiple of 32 ETH (1x, 2x, 3x, etc) - and automatically deploy more validators to the same pod.

I think these would be valuable additions, so will work on those now.

Reply:
Your question about having validators with other nonstandard balances is a good one though. So here's the idea: since you don't actually need to call `stake` to deposit and verify withdrawal credentials - we have a `userType` that skips that step in favor of depositing directly into the beacon chain.

Here's my what the changelog could look like, adding onto the changes I described above:

1. We change the `SIGNED_METHODS` usertype: it's now called `ALT_METHODS`. The concept is a user type that always calls the "non-default, but equivalent" methods.
2. We change `User_SignedMethods` (now called `User_AltMethods`) to override `depositIntoEigenlayer`. The new version does NOT call `eigenPodManager.stake`. Instead, it creates a pod, then directly deposits and verifies credentials. And this would mean it could handle other values for ETH (those supported by the deposit contract, anyway)
3. When dealing random assets, if we're handling `assetType == HOLDS_ETH` and `userType == ALT_METHODS`, we give them a random multiple of 32 ETH, but we ALSO add or subtract a random amount of gwei, e.g: { -10 gwei, -1 gwei, -0 gwei, +1 gwei, +10 gwei }

WDYT?

Reply:
`User_AltMethods` additions make sense to me. Is the reasoning of subtracting a random amount of gwei to test  `deposit value % 1 Gwei == 0` invariant from the deposit contract?